financialaid/views_test.py

```diff
@@ -498,6 +472,20 @@ def test_get_learner_price_for_course_allowed(self):
         expected_response = {
             "has_financial_aid_request": False,
             "course_price": self.course_price.price,
-            "financial_aid_adjustment": False
+            "financial_aid_adjustment": False,
+            "financial_aid_availability": True
+        }
+        self.assertDictEqual(resp.data, expected_response)
+        # Enrolled but program has no financial aid availability
+        self.program.financial_aid_availability = False
+        self.program.save()
+        resp = self.assert_http_status(self.client.get, self.course_price_url, status.HTTP_200_OK)
+        expected_response = {
+            "has_financial_aid_request": False,
+            "course_price": self.course_price.price,
+            "financial_aid_adjustment": False,
+            "financial_aid_availability": False
         }
         self.assertDictEqual(resp.data, expected_response)
+        self.program.financial_aid_availability = True
```

Comment:
The benefit is only for particular cases where you need to make changes that can't be reverted in a database transaction, for example `ESTestCase` does some Elasticsearch cleanup.

More information here: https://docs.djangoproject.com/en/1.10/topics/testing/tools/#django.test.TestCase.setUpTestData